schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+    id int PRIMARY KEY NOT NULL,
```

Comment:
- [ ] [OPTIONAL] You could improve this by making the id field not null and auto-incremented.
- [ ] [OPTIONAL] You could also make the id field the primary key of the table.